pkg/graph/accounts.go

```diff
@@ -22,26 +22,41 @@ import (
 
 	"github.com/arangodb/go-driver"
 	"github.com/slntopp/nocloud/pkg/nocloud"
+	"github.com/slntopp/nocloud/pkg/nocloud/access"
 	"github.com/slntopp/nocloud/pkg/nocloud/roles"
 	"github.com/slntopp/nocloud/pkg/nocloud/schema"
 	"go.uber.org/zap"
 
 	"github.com/slntopp/nocloud/pkg/credentials"
 	pb "github.com/slntopp/nocloud/pkg/registry/proto/accounts"
+
+	sc "github.com/slntopp/nocloud/pkg/settings/client"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
+const shouldCreateNamespaceKey = "should-create-namespace"
+
+type AccountSettings struct {
+	ShouldCreateNamespace bool `json:"ns"`
```

Comment:
then particullar setting must be like `should-create-ns`(json key must be appropriate)